pkg/ebpf/c/tracee.bpf.c

```diff
@@ -656,6 +664,345 @@ int tracepoint__sched__sched_process_fork(struct bpf_raw_tracepoint_args *ctx)
     return 0;
 }
 
+// If we won't create these maps, the userspace needs to be kernel version aware as well,
+// otherwise it'll try to interact with the maps and report an error
+BPF_HASH(unhidden_modules_map, u64, u64, MAX_NUM_MODS); // modules that are shown (not hidden)
+BPF_HASH(modules_from_procfs,
+         u64,
+         kernel_module_name_t,
+         MAX_NUM_MODS); // modules returned by reading /proc
+
+void __always_inline send_hidden_module(u64 mod_addr, char *mod_name, program_data_t *p)
+{
+    reset_event_args(p);
+    save_to_submit_buf(p->event, &mod_addr, sizeof(u64), 0);
+    save_bytes_to_buf(p->event, (void *) mod_name, MODULE_NAME_LEN & MAX_MEM_DUMP_SIZE, 1);
```

Comment:
It will be more clear to use `save_str_to_buf()` instead

Reply:
Unfortunately used the bytes helper while playing 1on1 with the verifier.. Perhaps I'll add a comment to clarify though